CONTRIBUTING.md

```diff
@@ -1,46 +1,32 @@
-# Contributing
+# Contributing to opentelemetry-dotnet
 
-## Report a bug or requesting feature
+The OpenTelemetry .NET special interest group (SIG) meets regularly. See the
+OpenTelemetry [community](https://github.com/open-telemetry/community#net-sdk)
+repo for information on this and other language SIGs.
 
-Reporting bugs is an important contribution. Please make sure to include:
+See the [public meeting notes](https://docs.google.com/document/d/1yjjD6aBcLxlRazYrawukDgrhZMObwHARJbB9glWdHj8/edit?usp=sharing)
+for a summary description of past meetings. To request edit access, join the
+meeting or get in touch on [Gitter](https://gitter.im/open-telemetry/opentelemetry-dotnet).
 
-- expected and actual behavior.
-- dotnet version that application is compiled on and running with (it may be
-  different - for instance target framework was set to .NET 4.6 for
-  compilation, but application is running on .NET 4.7.2).
-- exception call stack and other artifacts.
-- if possible - repro application and steps to reproduce.
+See to the [community membership document](https://github.com/open-telemetry/community/blob/master/community-membership.md)
+on how to become a [**Member**](https://github.com/open-telemetry/community/blob/master/community-membership.md#member),
+[**Approver**](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver)
+and [**Maintainer**](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer).
 
-## How to contribute
+# Find a Buddy and get Started Quickly!
 
-### Before you start
+If you are looking for someone to help you find a starting point and be a resource for your first contribution, join our
+Gitter and find a buddy!
 
-Please read project contribution
-[guide](https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md)
-for general practices for OpenTelemetry project.
+1. Join [Gitter.im](https://gitter.im) and join our [chat room](https://gitter.im/open-telemetry/opentelemetry-dotnet).
+2. Post in the room with an introduction to yourself, what area you are interested in (check issues marked "Help Wanted"),
```

Comment:
"good first issue" and "up-for-grabs" are the labels currently used. Either use that here, or lets cleanup other labels to match "help wanted"

Reply:
I've added links.
I think we should use "good first issue" and "help wanted" since they are standard GitHub tags (which GitHub would give you by default when you create a new repo).

Reply:
renamed